tests/FakeItEasy.Tests/Configuration/RuleBuilderTests.cs

```diff
@@ -23,7 +23,7 @@ public RuleBuilderTests()
             this.asserter = A.Fake<IFakeAsserter>();
             this.ruleProducedByFactory = A.Fake<BuildableCallRule>();
 
-            this.fakeManager = A.Fake<FakeManager>(o => o.CallsBaseMethods());
+            this.fakeManager = new FakeManager(typeof(object), new object());

```

Comment:
In case you were wondering about this: this is because if we fake the `FakeManager`, `FakeObjectType` returns a fake type on which `GetTypeInfo()` throws an exception. We don't actually need it to be a fake, since we don't configure any call on it.

Reply:
I was mildly curious, but you had me at "one less fake".